src/Plugin/facets/query_type/LocalGovDirectoriesQueryType.php

```diff
@@ -147,7 +147,7 @@ protected function getResultingFacetsFromFacetGroupExceptOwn(string $filter_tag)
     $filter_query->preExecute();
 
     // Find conditions.
-    $conditions = &$filter_query->getConditionGroup()->getConditions();
+    $conditions = $filter_query->getConditionGroup()->getConditions();
```

Comment:
My finding is we need the `&` and take this by reference or we get strange behaviour where the conditions don't get reset.
To test, I added countries to the demo Money, employment and training directory.

Without taking the query by reference.
<img width="904" alt="Screenshot 2023-10-16 at 4 59 19 pm" src="https://github.com/localgovdrupal/localgov_directories/assets/1467480/eb060238-e18a-44ea-aa2b-1c672762648f">

With query by reference
<img width="856" alt="Screenshot 2023-10-17 at 10 37 25 am" src="https://github.com/localgovdrupal/localgov_directories/assets/1467480/fcae3e37-8dd9-471c-a16d-3a11847228c4">

Reply:
Note: Northern ireland should still be visible even when not ticked.

<img width="988" alt="Screenshot 2023-10-16 at 4 59 37 pm" src="https://github.com/localgovdrupal/localgov_directories/assets/1467480/433d5bd0-2d89-4d63-8a79-4ed8a5380d0c">